### DISCUSSION TITLE
Download URLs from file

### DISCUSSION THREAD

**Discussion by Brainfighter** [crunchy-labs/crunchy-cli#280] [Ideas]
Is it possible to load series URLs from a file so that I can include multiple series to download ?

**Comment by bytedream** [comment]
There is no support for this in the tool itself, but you can use workaround relatively easy.

E.g. given the file `series.txt`:
```
https://www.crunchyroll.com/series/GY8VEQ95Y/darling-in-the-franxx
https://www.crunchyroll.com/series/GG5H5XQX4/frieren-beyond-journeys-end
https://www.crunchyroll.com/series/GY2P9ED0Y/code-geass
```

On linux you can `cat` the file which works perfectly fine as inputas the cli supports multiple urls:
```shell
$ crunchy-cli download $(cat series.txt)
```
On Windows it should be similar, but I do not use Windows active so I don't know the correct command
